src/app/login/page.tsx

```diff
@@ -12,75 +12,73 @@ import { useEffect, useState } from "react";
 const changeEventHandlerFactory =
   (setter: React.Dispatch<React.SetStateAction<string>>) =>
   (e: React.ChangeEvent<HTMLInputElement>) => {
-    setter(e.target.value);
-  };
+    setter(e.target.value)
+  }
 
 const Login = () => {
-  const router = useRouter();
+  const router = useRouter()
 
-  const [email, setEmail] = useState("");
-  const [password, setPassword] = useState("");
-  const [loginStatus, setLoginStatus] = useState<"success" | "error">();
+  const [email, setEmail] = useState("")
+  const [password, setPassword] = useState("")
+  const [loginStatus, setLoginStatus] = useState<"success" | "error">()
```

Comment:
initial state?

Reply:
Yes, I initialized it as I used them later on in the code for managing form input and login status. They were initially set to empty strings for (email and password) and for (loginStatus) it can be one of the two: success or error.

Reply:
Since I didn't provide an initial value aka (), so its initial state is undefined.